chemprop/train/make_predictions.py

```diff
@@ -122,6 +123,7 @@ def predict_and_save(args: PredictArgs, train_args: TrainArgs, test_data: Molecu
     :param full_to_valid_indices: A dictionary dictionary mapping full to valid indices.
     :param models: A list or generator object of :class:`~chemprop.models.MoleculeModel`\ s.
     :param scalers: A list or generator object of :class:`~chemprop.features.scaler.StandardScaler` objects.
+    :param return_invalid_smiles: Whether to include invalid SMILES with a value None in the returned predictions.
```

Comment:
I probably misunderstood, but it looks like the description says that a `None` value is returned, when in practice an `'Invalid SMILES'` string is returned

Reply:
You are completely right here. This is a mistake in the docstring and I will fix.